rllib/env/single_agent_env_runner.py

```diff
@@ -41,56 +41,48 @@ def __init__(self, config: "AlgorithmConfig", **kwargs):
         # Get the worker index on which this instance is running.
         self.worker_index: int = kwargs.get("worker_index")
 
+        # Create the vectorized gymnasium env.
+
         # Register env for the local context.
         # Note, `gym.register` has to be called on each worker.
-        gym.register(
-            "custom-env-v0",
-            partial(
+        if (
```

Comment:
This is a bug fix. Otherwise, passing in a class to `config.environment(env=[some class])` does not work (only strings work).